src/services/selfHosted/SelfHostedUserListQueryOpts.ts

```diff
@@ -51,32 +50,24 @@ export const useSelfHostedUserList = ({
         const parsedData = RequestSchema.safeParse(res)
 
         if (!parsedData.success) {
-          return Promise.reject({
+          return rejectNetworkError({
             status: 404,
             data: {},
-            dev: 'useSelfHostedUserList - 404 schema parsing failed',
-          } satisfies NetworkErrorObject)
+            dev: 'SelfHostedUserListQueryOpts - 404 schema parsing failed',
+            error: parsedData.error,
+          })
         }
+
         return parsedData.data
       })
     },
-    suspense: false,
+    initialPageParam: '1',
```

Comment:
is this not defaulted to 1 already?

Reply:
It is not, you have to explicitly define it in V5